### PR TITLE
Updating Swagger UI and changed resource registrations

### PR DESCRIPTION
**Pull request by dchesterman**
Updated Swagger UI and also fixed an issue that I was seeing where Swagger UI was throwing an error due to invalid swagger.json being generated.  Basically all fields were present with null values which didn't pass the Swagger Spec Validator.  Gist of what I was getting before : https://gist.github.com/dchesterman/775dccc70d32d813b4ff

### CODE REVIEW

File: src/main/java/io/federecio/dropwizard/swagger/SwaggerBundle.java [hunk 1]
```diff
@@ -60,10 +58,10 @@ public void run(T configuration, Environment environment) throws Exception {
         new AssetsBundle(Constants.SWAGGER_RESOURCES_PATH, configurationHelper.getSwaggerUriPath(), null, Constants.SWAGGER_ASSETS_NAME).run(environment);
 
         environment.jersey().register(new SwaggerResource(configurationHelper.getUrlPattern()));
-        environment.getObjectMapper().setSerializationInclusion(JsonInclude.Include.NON_NULL);
```
Comment (dchesterman) [inline src/main/java/io/federecio/dropwizard/swagger/SwaggerBundle.java]: I believe this is what was causing the issue I was seeing.  Do you get valid swagger specs back the way it currently is?  Thanks for forking this.

**Comment by dchesterman** [comment]
Cool. Ty.  Any chance this could be pushed to central?

**Comment by akraxx** [comment]
:thumbsup:

**Comment by jplock** [comment]
I just pushed `0.7.2` to maven central.  Should show up in a few hours.

**Pull request merged**